docs/introduction.rst

```diff
@@ -99,107 +99,127 @@ The pywbemtools package is supported in these environments:
+and ``pywbemlistener`` commands available in a directory that is in the PATH.
+These commands will be available that way, regardless of whether or not you
+have a virtual Python environment active (that you may need for other purposes).
 
+1.  Prerequisite: Install pipx as an OS-level package
 
-.. _`Installation prerequisites`:
+    Follow the steps at https://pipx.pypa.io/stable/installation/ to install
+    pipx as an OS-level package to your local system.
 
-Installation prerequisites
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+2.  Without having any virtual Python environment active, install pywbemtools using
+    pipx
 
-The Python environment into which you want to install must have the following
-Python packages installed:
+    To install the latest released version of pywbemtools:
 
-- setuptools - http://pypi.python.org/pypi/setuptools
-- wheel
-- pip
+    .. code-block:: bash
 
-Pywbemtools installs the pywbem package.
+        $ pipx install pywbemtools
 
-When using pywbem versions before 1.0.0 on Python 2, pywbem requires a number
-of OS-level packages, and your system must have the following commands
-installed:
+    To install a specific released version of pywbemtools, e.g. 1.2.0:
 
-* On native Windows:
+    .. code-block:: bash
 
-  - ``choco`` - Chocolatey package manager. The pywbemtools package installation
-    uses Chocolatey to install OS-level software. See https://chocolatey.org/
-    for the installation instructions for Chocolatey.
+        $ pipx install pywbemtools==1.2.0
 
-  - ``wget`` - Download tool. Can be installed with: ``choco install wget``.
+    To install a specific development branch of pywbemtools, e.g. master:
 
-* On Linux, OS-X, UNIX-like environments on Windows (e.g. Cygwin):
+    .. code-block:: bash
 
-  - ``wget`` - Download tool. Can be installed using the OS-level package
-    manager for the platform.
+        $ pipx install git+https://github.com/pywbem/pywbemtools.git@master
 
-.. _`Installation with pip`:
+    To install pywbemtools with a non-default Python version, e.g. 3.10:
 
-Installation with pip
-^^^^^^^^^^^^^^^^^^^^^
+    .. code-block:: bash
 
-When using pywbem versions before 1.0.0 on Python 2, install the OS-level
-packages needed by the pywbem package as follows:
+        $ pipx install pywbemtools --python python3.10
 
-* On native Windows:
+Installation into a virtual Python environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  .. code-block:: bash
+In some cases it may be useful to install pywbemtools into your own
+`virtual Python environment`_. That avoids the dependency to pipx, but it
+requires you to activate the virtual environment every time you want to use the
+``pywbemcli`` or ``pywbemlistener`` commands.
 
-      > wget -q https://raw.githubusercontent.com/pywbem/pywbem/master/pywbem_os_setup.bat
-      > pywbem_os_setup.bat
+There is a number of ways how virtual Python environments can be created. This
```

Comment:
minor grammar.  "There are a number of ways virtual Python environments ...

changes is to are and drop "how"

Reply:
DONE